samples/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/domain/ShoppingCartImpl.java

```diff
@@ -14,69 +14,53 @@
  * limitations under the License.
  */
 
-package eventsourced.shoppingcart;
+package com.example.shoppingcart.domain;
 
 import com.akkaserverless.javasdk.EntityId;
-import com.akkaserverless.javasdk.eventsourcedentity.*;
+import com.akkaserverless.javasdk.eventsourcedentity.CommandContext;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntity;
 import com.example.shoppingcart.ShoppingCartApi;
-import com.example.shoppingcart.domain.ShoppingCartDomain;
 import com.google.protobuf.Empty;
 
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.stream.Collectors;
 
-/** An event sourced entity. */
+/**
+ * An event sourced entity.
+ */
+
+/**
+ * An event sourced entity.
+ */
```

Comment:
Useless comments become better if you have them duplicated!

Reply:
I'm glad you noticed! :-)